--- src/libjson/JSONOptions.h	2012-03-10 17:41:30.000000000 +0800
+++ patch/libjson/JSONOptions.h	2012-08-26 20:42:55.000000000 +0800
@@ -28,7 +29,7 @@
  *  it simply tells you about them, which is nice for debugging, but not preferable
  *  for release candidates
  */
-//#define JSON_DEBUG
+#define JSON_DEBUG
 
 
 /*
@@ -36,6 +37,7 @@
  *  references to long long and long double as well as a few others
  */
 //#define JSON_ISO_STRICT
+#define JSON_ISO_STRICT
 
 
 /*
@@ -51,7 +53,7 @@
  *  option hides the callback registering function.  This will usually display
  *  messages in the console
  */
-//#define JSON_STDERROR
+#define JSON_STDERROR
 
 
 /*
@@ -94,7 +96,7 @@
  *  JSON_BINARY is used to support binary, which is base64 encoded and decoded by libjson,
  *  if this option is not turned off, no base64 support is included
  */
-#define JSON_BINARY
+//#define JSON_BINARY
 
 
 /*
@@ -102,7 +104,7 @@
  *  and decoding.  This may be useful if you want to obfuscate your json, or send binary data over
  *  a network
  */
-#define JSON_EXPOSE_BASE64
+//#define JSON_EXPOSE_BASE64
 
 
 /*
@@ -321,7 +323,7 @@
  *  compatibility between major releases.  It is highly recommended that you move your functions
  *  over to the new equivalents
  */
-#define JSON_DEPRECATED_FUNCTIONS
+//#define JSON_DEPRECATED_FUNCTIONS
 
 
 /*
